2021-05-28 22:19:01,938 INFO    MainThread:21128 [wandb_setup.py:_flush():69] setting env: {}
2021-05-28 22:19:01,938 INFO    MainThread:21128 [wandb_setup.py:_flush():69] setting login settings: {}
2021-05-28 22:19:01,938 INFO    MainThread:21128 [wandb_init.py:_log_setup():334] Logging user logs to C:\Users\user\DAN Lab\제주도가즈아\wandb\run-20210528_221901-35z73rzz\logs\debug.log
2021-05-28 22:19:01,938 INFO    MainThread:21128 [wandb_init.py:_log_setup():335] Logging internal logs to C:\Users\user\DAN Lab\제주도가즈아\wandb\run-20210528_221901-35z73rzz\logs\debug-internal.log
2021-05-28 22:19:01,938 INFO    MainThread:21128 [wandb_init.py:init():367] calling init triggers
2021-05-28 22:19:01,938 INFO    MainThread:21128 [wandb_init.py:init():372] wandb.init called with sweep_config: {}
config: {}
2021-05-28 22:19:01,939 INFO    MainThread:21128 [wandb_init.py:init():416] starting backend
2021-05-28 22:19:01,939 INFO    MainThread:21128 [backend.py:_multiprocessing_setup():69] multiprocessing start_methods=spawn, using: spawn
2021-05-28 22:19:01,941 INFO    MainThread:21128 [backend.py:ensure_launched():123] starting backend process...
2021-05-28 22:19:02,013 INFO    MainThread:21128 [backend.py:ensure_launched():127] started backend process with pid: 19664
2021-05-28 22:19:02,014 INFO    MainThread:21128 [wandb_init.py:init():421] backend started and connected
2021-05-28 22:19:02,014 INFO    MainThread:21128 [wandb_init.py:init():461] updated telemetry
2021-05-28 22:19:02,015 INFO    MainThread:21128 [wandb_init.py:init():480] communicating current version
2021-05-28 22:19:03,267 INFO    MainThread:21128 [wandb_init.py:init():485] got version response upgrade_message: "wandb version 0.10.31 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2021-05-28 22:19:03,268 INFO    MainThread:21128 [wandb_init.py:init():493] communicating run to backend with 30 second timeout
2021-05-28 22:19:03,584 INFO    MainThread:21128 [wandb_init.py:init():518] starting run threads in backend
2021-05-28 22:19:05,605 INFO    MainThread:21128 [wandb_run.py:_console_start():1476] atexit reg
2021-05-28 22:19:05,606 INFO    MainThread:21128 [wandb_run.py:_redirect():1339] redirect: SettingsConsole.WRAP
2021-05-28 22:19:05,606 INFO    MainThread:21128 [wandb_run.py:_redirect():1374] Wrapping output streams.
2021-05-28 22:19:05,606 INFO    MainThread:21128 [wandb_run.py:_redirect():1390] Redirects installed.
2021-05-28 22:19:05,606 INFO    MainThread:21128 [wandb_init.py:init():542] run started, returning control to user process
2021-05-28 22:40:27,148 INFO    MainThread:21128 [wandb_run.py:_atexit_cleanup():1446] got exitcode: 1
2021-05-28 22:40:27,148 INFO    MainThread:21128 [wandb_run.py:_restore():1418] restore
2021-05-28 22:40:27,423 INFO    MainThread:21128 [wandb_run.py:_wait_for_finish():1568] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 538
  total_bytes: 538
}

2021-05-28 22:40:29,427 INFO    MainThread:21128 [wandb_run.py:_wait_for_finish():1568] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 578819
  total_bytes: 578819
}

2021-05-28 22:40:30,699 INFO    MainThread:21128 [wandb_run.py:_show_summary():1723] rendering summary
2021-05-28 22:40:30,701 INFO    MainThread:21128 [wandb_run.py:_show_history():1761] rendering history
2021-05-28 22:40:30,703 INFO    MainThread:21128 [wandb_run.py:_show_files():1790] logging synced files
